src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -409,6 +423,14 @@ public override void VisitMethod(IMethodSymbol symbol)
             switch (symbol.MethodKind)
             {
                 case MethodKind.Ordinary:
+                    var symbolKind = symbol.IsStatic
+                        ? SymbolDisplayPartKind.StaticMethodName
+                        : SymbolDisplayPartKind.MethodName;
```

Comment:
This is wrong approach to fixing this problem and it will break VS for Mac, because we expect `SymbolDisplayPartKind.MethodName` to be defined for methods.
Correct way to do this is, to set only `SymbolDisplayPartKind.MethodName` for `instance` methods and both `SymbolDisplayPartKind.MethodName` and `SymbolDisplayPartKind.StaticMethodName` for `static` methods and set correct Order Priority or BaseTypes so `SymbolDisplayPartKind.StaticMethodName` has priority over `SymbolDisplayPartKind.MethodName`, so if it's defined it's color is taken over normal method color, so if `SymbolDisplayPartKind.StaticMethodName` color is not defined in theme, `SymbolDisplayPartKind.MethodName` should be respected for `static` methods as is the case in VS for Mac atm.

Reply:
SymbolDisplayPartKind don't have an inheritance system.  The ClassificationTypeDefinitions already have inheritance setup in this way you describe and I could update the ClassificationFormatDefinitions to explicitly have the ordering.

Reply:
You are right, I missed the fact that `BaseDefinition` is correctly set and that it will automatically set `SymbolDisplayPartKind.MethodName` when `SymbolDisplayPartKind.StaticMethodName` is defined. Sorry about noise.

Reply:
No worries. Thanks for taking a look. Would love to see the 2019 enhanced Roslyn colors make it into VS for Mac. I took a stab at creating the themes (https://gist.github.com/JoeRobich/c108f8338d2bc0b277dc147a778be2ab) but they are only ~90% complete.

Reply:
What is 10% that is missing?

Reply:
Things outside the editor mostly, message bubbles for example, are left the same colors as the theme I I based them on. I think the experience with the themes is very close as they are.

Also, there is a mismatch with classifications and I was unable to get context keywords to be the new purple colors when the editor was using semantic classifications. Not sure which of the MonoDevelop `Keyword(*)` classifications match up to Roslyn's `Keyword - Control`.